### ISSUE TITLE
glibc 2.35 breaks libTAS

### ISSUE THREAD

**Issue by marcelopio** [clementgallet/libTAS#471]
Arch Linux updated to glibc 2.35 and it's package do not expose libc_thread_freeres
`/home/belfalas/.local/share/Steam/steamapps/common/Dandara/Dandara.x86_64: symbol lookup error: /usr/bin/libtas.so: undefined symbol: __libc_thread_freeres`

Maybe because of this change:
https://github.com/bminor/glibc/commit/f47f1d91af985a9028fb399da21eab460d887a15

**Comment by clementgallet** [comment]
Yeah, it should be fixed in commit 57398a5b9f094d57b4b52adc8b45d7937a779528
